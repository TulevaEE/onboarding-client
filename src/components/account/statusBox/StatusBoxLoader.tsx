import React from 'react';
import { Shimmer } from '../../common/shimmer/Shimmer';
import { StatusBoxTitle } from './StatusBoxTitle';

export const StatusBoxLoader: React.FunctionComponent = () => (
  <>
    <StatusBoxTitle />
    <div className="card card-secondary">
      <div className="d-flex p-2 status-box-row tv-table__row">
        <Shimmer height={52} />
      </div>
      <div className="d-flex p-2 status-box-row tv-table__row">
        <Shimmer height={76} />
      </div>
      <div className="d-flex p-2 status-box-row">
        <Shimmer height={52} />
      </div>
    </div>
  </>
);
