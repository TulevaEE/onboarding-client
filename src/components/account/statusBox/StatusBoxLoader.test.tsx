import React from 'react';
import { shallow } from 'enzyme';
import { StatusBoxLoader } from './StatusBoxLoader';

describe('StatusBoxLoader', () => {
  const component = shallow(<StatusBoxLoader />);

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});
