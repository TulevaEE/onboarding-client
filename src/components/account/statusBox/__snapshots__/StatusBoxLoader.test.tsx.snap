// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBoxLoader renders correctly 1`] = `
<Fragment>
  <StatusBoxTitle />
  <div
    className="card card-secondary"
  >
    <div
      className="d-flex p-3 status-box-row tv-table__row"
    >
      <Shimmer
        height={52}
      />
    </div>
    <div
      className="d-flex p-3 status-box-row tv-table__row"
    >
      <Shimmer
        height={76}
      />
    </div>
    <div
      className="d-flex p-3 status-box-row"
    >
      <Shimmer
        height={76}
      />
    </div>
  </div>
</Fragment>
`;
